### PR TITLE
feat: add firebase functionality

### PR DESCRIPTION
**Pull request by fernandogelin**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ if (IS_ELECTRON) {
   psiturk = new PsiTurk(getTurkUniqueId(), '/complete')
   /* eslint-enable */
 }
+const firebase = true;
```
Comment (mcmcgrath13) [inline src/App.js]: how can this get overwritten to false? should there be an env var?

File: src/App.js [hunk 2]
```diff
@@ -29,6 +31,16 @@ class App extends React.Component {
         <Experiment settings={{
           timeline: tl,
           on_data_update: (data) => {
+            if (firebase) {
+              if (data.trial_index === 1 ) {
```
Comment (mcmcgrath13) [inline src/App.js:35]: isn't there an issue sometimes where we don't have the patient ID on the first trial? and don't get it until a few screens in?

File: src/App.js (hunk 1 above)
Comment (fernandogelin) [reply src/App.js]: I think I changed that in the latest commit.. maybe I didn't push?

File: src/App.js (hunk 2 above)
Comment (fernandogelin) [reply src/App.js:35]: first trial is 0. 1 has the patient id. I guess it depends on the task.

File: src/App.js (hunk 1 above)
Comment (fernandogelin) [reply src/App.js]: I did push.. maybe the PR didn't update :/

Comment (fernandogelin) [reply src/App.js]: it says `outdated` here

**Comment by mcmcgrath13** [comment]
I'd make it more abstract with a boolean for `dbInitialized` and then once
we get a trial with a `patient_id` do that and turn it to true

**Comment by fernandogelin** [comment]
Actually no, what needs to change is the function that creates the db document. If it doesn't have a patient id it can still create documents with randomly generated IDs. We don't need to merge this right now, it's mostly to show Wasita what changes need to be made.

File: src/App.js [hunk 3]
```diff
@@ -48,7 +60,8 @@ class App extends React.Component {
               }
               completePsiturk()
             } else {
-              jsPsych.data.get().localSave('csv','neuro-task.csv');
+              
+              // jsPsych.data.get().localSave('csv','neuro-task.csv');
```
Comment (mcmcgrath13) [inline src/App.js]: why comment this out?

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**